aws-replicator/aws_replicator/client/cli.py

```diff
@@ -78,7 +78,11 @@ def cmd_aws_proxy(services: str, config: str, container: bool, port: int, host:
 
     config_json: ProxyConfig = {"services": {}}
     if config:
-        config_json = yaml.load(load_file(config), Loader=yaml.SafeLoader)
+        loaded_file = load_file(config)
+        if not loaded_file:
+            console.print(f"Config file {config} does not exist")
+            sys.exit(1)
```

Comment:
nit: Probably better to use `https://click.palletsprojects.com/en/8.1.x/api/#click.Context.exit` here, so `click` can run any cleanup/shutdown logic, if any.
```
...
@click.pass_context
def cmd_aws_proxy(context, services: str, config: str, container: bool, port: int, host: str):
    from aws_replicator.client.auth_proxy import start_aws_auth_proxy_in_container
    
    if ...:
        ...
        context.exit(2)

```